src/common.js

```diff
@@ -96,6 +96,44 @@ var ShellString = function (stdout, stderr, code) {
   } else {
     that = new String(stdout);
     that.stdout = stdout;
+
+    that.forEach = function (callback, opts) {
```

Comment:
In this case, I am adding a .forEach method to that, where that is a String(), which doesn't have a .forEach method by default. In the case of ls(), that is an Array, so I don't override the forEach method.